docs/rebuild_admin.rst

```diff
@@ -242,85 +238,78 @@ In addition to the account and networking information retrieved from the servers
 so far, you'll need to retrieve the following files and info:
 
  - Tor Onion Service URLs and tokens
- - GPG *Submission Public Key* and OSSEC public key
+ - GPG *Submission Public Key*, *OSSEC Alert Public Key*, and (optional)
+   *Journalist Alert Public Key*
  - OSSEC alert configuration details
  - (Optional) HTTPS configuration details
 
-Retrieve Onion Service info
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
+Retrieve Onion Service Info
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 In order to connect to the onion services used by your instance, you will
 need to copy their details to the *Admin Workstation*. To do so, run the following
 commands from a Terminal window:
 
 .. code:: sh
 
  cd ~/Persistent/securedrop/install_files/ansible-base
- echo "HidServAuth $(ssh app sudo cat /var/lib/tor/services/journalist/hostname)" \
-    > app-journalist-aths
- echo "$(ssh app sudo cat /var/lib/tor/services/source/hostname)" > app-source-ths
-
-If your instance was set up to use SSH over TOR, you'll also need to copy over the details
-of the SSH proxy onion services, by running the following commands:
-
-.. code:: sh
-
- echo "HidServAuth $(ssh app sudo cat /var/lib/tor/services/ssh/hostname)" > app-ssh-aths
- echo "HidServAuth $(ssh mon sudo cat /var/lib/tor/services/ssh/hostname)" > mon-ssh-aths
+ echo "$(ssh app sudo cat /var/lib/tor/services/journalistv3/hostname)" \
+    > app-journalistv3-ths
```

Comment:
This file isn't used (see comment above). The server doesn't have the private key necessary to build the `app-*.auth_private` files needed by `tailsconfig` to set up access.

Reply:
Right, thanks, this is unnecessary, and I've removed it in 33f6c42